x-pack/plugins/enterprise_search/public/applications/shared/licensing/manage_license_button.tsx

```diff
@@ -27,12 +27,7 @@ export const ManageLicenseButton: React.FC<EuiButtonProps> = (props) => {
       })}
     </EuiButtonTo>
   ) : (
-    <EuiButton
-      {...props}
-      target="_blank"
-      iconType="popout"
-      href={`${docLinks.enterpriseSearchBase}/license-management.html`}
-    >
+    <EuiButton {...props} target="_blank" iconType="popout" href={`${docLinks.licenseManagement}`}>
```

Comment:
```suggestion
    <EuiButton {...props} target="_blank" iconType="popout" href={docLinks.licenseManagement}>
```
No need for a template literal here